data/json/items/armor/holster.json

```diff
@@ -233,20 +233,32 @@
   },
   {
     "id": "western_holster",
-    "copy-from": "holster",
     "type": "ARMOR",
-    "name": { "str": "western holster" },
-    "description": "A Western-style holster as worn by cowboys and gunfighters in the movies and on TV. Has loops across the back to hold spare ammo.  Activate to holster/draw a gun.",
+    "name": { "str": "western gun belt" },
+    "description": "A Western-style gun belt as worn by cowboys and gunfighters in the movies and on TV. Has loops across the back to hold spare ammo.  Activate to holster/draw a gun.",
     "weight": "260 g",
     "volume": "550 ml",
     "price_postapoc": 500,
     "material": [ "leather" ],
+    "symbol": "[",
+    "looks_like": "sheath",
+    "color": "light_gray",
     "sided": true,
+    "material_thickness": 1,
     "armor": [
-      { "covers": [ "torso" ], "coverage": 5, "encumbrance": [ 1, 1 ], "specifically_covers": [ "torso_lower" ] },
+      {
+        "covers": [ "torso" ],
+        "coverage": 50,
+        "specifically_covers": [ "torso_waist", "torso_lower" ],
+        "encumbrance": [ 1, 1 ],
+        "material": [
+          { "type": "leather", "covered_by_mat": 100, "thickness": 0.75 },
+          { "type": "steel", "covered_by_mat": 100, "thickness": 0.75 }
```

Comment:
There's no steel mentioned in either the description or the item material list.  The coverage provides 50% more thickness than the item itself.  I would have thought that would trigger a test failure, but maybe I'm misunderstanding.

Reply:
its the belt buckle from the leather belt